RELEASING.md

```diff
@@ -95,64 +101,23 @@ Go to the PR created and:
 
 1. Create a git tag for the new commit on the release branch and push it:
 
-```
-git tag v1.30.3
-git push --tags
-```
+    ```sh
+    git tag v1.30.3
+    git push --tags
+    ```
 
 1. Log in to semaphore and find the new build for the release branch commit, and
    click 'Rerun'. When Semaphore starts the rebuild, it will notice the new tag and
    build and publish an operator release.
 
-## Release notes and milestones
-
-1. Run the following command to generate release notes for the release
-
-   ```
-   make release-notes VERSION=<TAG> GITHUB_TOKEN=<access-token>
-   ```
+1. Go to [releases](https://github.com/tigera/operator/releases) and edit the draft release for the release tag
 
-1. Go to https://github.com/tigera/operator/releases and edit the release tag to include the generated release notes, and update the title.
+1. Publish the release.
 
-1. Close the milestone for this release. https://github.com/tigera/operator/milestones
```

Comment:
This still must be done AFAICT, don't we need to move this before doing the 'Rerun' step above or else the Rerun won't create the release draft?

Reply:
The release notes now require that the milestone is closed first before generating the draft release with release notes.